op.c

```diff
@@ -16171,14 +16171,22 @@ Perl_coresub_op(pTHX_ SV * const coreargssv, const int code,
     PERL_ARGS_ASSERT_CORESUB_OP;
 
     switch(opnum) {
-    case 0:
+    case 0: {
+        int caller_index = -1;
+        switch (-code) {
+            case (KEY___PACKAGE__): { caller_index = 0; break; } /* (caller)[0] */
+            case (KEY___FILE__):    { caller_index = 1; break; } /* (caller)[1] */
+            case (KEY___LINE__):    { caller_index = 2; break; } /* (caller)[2] */
```

Comment:
Why all the extra brackets? Why not just:
```suggestion
            case KEY___PACKAGE__: caller_index = 0; break; /* (caller)[0] */
            case KEY___FILE__:    caller_index = 1; break; /* (caller)[1] */
            case KEY___LINE__:    caller_index = 2; break; /* (caller)[2] */
```

Reply:
it's just habit, mostly driven by paired characters highlighting feature of editors.

Reply:
74afe9ddf4